### PR TITLE
Clicking a bitrunning pod orbits its avatar

### PR DESCRIPTION
**Pull request by Rhials**
## About The Pull Request

Clicking an active bitrunner pod as an observer will allow you to orbit that bitrunning pod's current avatar.
## Why It's Good For The Game

Makes it easier to check up on what those cheeky Bitrunners are up to!
## Changelog
:cl: Rhials
qol: As an observer, clicking on a bitrunning pod will let you orbit it's bitrunning avatar. Cool!
/:cl:

### CODE REVIEW

File: code/modules/bitrunning/objects/netpod.dm [hunk 1]
```diff
@@ -227,6 +227,11 @@
 
 	return FALSE
 
+/obj/machinery/netpod/attack_ghost(mob/dead/observer/our_observer)
+	var/our_target = avatar_ref?.resolve()
+	if(!our_target || our_observer.orbit(our_target))
```
Comment (jlsnow301) [inline code/modules/bitrunning/objects/netpod.dm:232]: Use `isnull()` please

**Comment by nikothedude** [comment]
Was this tested with revenants?

**Comment by jlsnow301** [comment]
> Was this tested with revenants?

i sense an issue report coming

**Comment by Rhials** [comment]
> Was this tested with revenants?

Revenants are living mobs, and shouldn't be calling attack_ghost with their clicks. I haven't tested it but I can say with some degree of confidence that it won't cause problems.

**Pull request merged**